14-minimal/root/opt/app-root/etc/generate_container_user

```diff
@@ -0,0 +1,17 @@
+# Set current user in nss_wrapper
+USER_ID=$(id -u)
+GROUP_ID=$(id -g)
+
+if [ x"$USER_ID" != x"0" -a x"$USER_ID" != x"1001" ]; then
```

Comment:
```suggestion
if [ x"$USER_ID" != x"0" ] && [ x"$USER_ID" != x"1001" ]; then
```

Reply:
This will be present in other versions as well, so I would rather follow up with a different PR.